nixos/modules/services/backup/restic.nix

```diff
@@ -256,6 +256,24 @@ in
             having to manually specify most options.
           '';
         };
+
+        niceness = mkOption {
+          description = lib.mdDoc "Niceness for instances of restic. Also applies to remote ones connecting via ssh when positive. Only used for `restic backup`, `restic forget` and `restic check` commands.";
+          type = types.ints.between (-20) 19;
+          default = 10;
```

Comment:
This changes the current behavior and would give restic way more CPU priority, resulting in a surprising change for some. Thus, I would set the `default` value to the default `nice` value of 0.

Reply:
From https://man7.org/linux/man-pages/man2/setpriority.2.html:

> The prio argument is a value in the range -20 to 19 (but see NOTES below), with -20 being the highest priority and 19 being the lowest priority.  Attempts to set a priority outside this range are silently clamped to the range.  The default priority is 0; lower values give a process a higher scheduling priority.

I agree we should have 0 as the default, if anything just to be backwards compatible. But isn't it the opposite - a higher value means _less_ CPU priority?

Reply:
>But isn't it the opposite - a higher value means _less_ CPU priority?

Yes, of course you are right. I was confused.

Reply:
np, it's pretty new to me so I wanted to make sure.